src/coreclr/vm/jithelpers.cpp

```diff
@@ -5518,8 +5520,9 @@ void F_CALL_CONV HCCALL1(JIT_ReversePInvokeExitTrackTransitions, ReversePInvokeF
     }
 #endif
 }
+HCIMPLEND_RAW
 
-void F_CALL_CONV HCCALL1(JIT_ReversePInvokeExit, ReversePInvokeFrame* frame)
+HCIMPL1(void, JIT_ReversePInvokeExit, ReversePInvokeFrame* frame)
```

Comment:
This should also be `HCIMPL1_RAW` like the other exit helper.

Reply:
Thanks!